src/styles/fonts.ts

```diff
@@ -135,7 +135,36 @@ export const fontInfoMap: Record<string, FontInfo[]> = {
         },
     ],
 };
+export const PdfFontType = Object.freeze({
```

Comment:
```suggestion
export const pdfFonts = Object.freeze({
```

type 정의가 아니면 카멜케이스로 정의합니다.
type이 아니므로 Type postfix가 없으면 더 좋을 것 같습니다!